freqtrade/freqai/data_drawer.py

```diff
@@ -74,8 +74,9 @@ def __init__(self, full_path: Path, config: Config):
         self.historic_predictions: Dict[str, DataFrame] = {}
         self.full_path = full_path
         self.historic_predictions_path = Path(self.full_path / "historic_predictions.pkl")
-        self.historic_predictions_bkp_path = Path(
-            self.full_path / "historic_predictions.backup.pkl")
+        self.historic_predictions_folder = Path(self.full_path / "historic_predictions")
+        self.historic_predictions_bkp_folder = Path(
+            self.full_path / "historic_predictions_backup")
```

Comment:
Nitpick: fits onto one line 

```suggestion
        self.historic_predictions_bkp_folder = Path(self.full_path / "historic_predictions_backup")
```